src/pages/index.tsx

```diff
@@ -161,6 +163,7 @@ export const Home = () => {
             >
               <Box
                 component='img'
+                alt='hero background image'
```

Comment:
<img width="649" alt="alt" src="https://user-images.githubusercontent.com/56411016/219798861-8157e96c-f768-46ef-b971-b329371ca17e.png">

Reply:
`image` isn't needed since this has a role of `img` (since it's an `img` element). Again though, this should probably be `aria-hidden={true}` and taken out of the accessibility tree.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden